### PR TITLE
Issue 14047: Clean up of Jakarta mail

### PR DESCRIPTION
**Pull request by uberskigeek**
Small changes for Jakarta mail

### CODE REVIEW

File: dev/com.ibm.websphere.appserver.features/visibility/public/mail-2.0/resources/l10n/io.openliberty.mail-2.0.properties [hunk 1]
```diff
@@ -14,4 +14,4 @@
 #NLS_MESSAGEFORMAT_NONE
 #
 
-description=This feature allows applications to utilize the JakartaMail 2.0 API. 
+description=This feature allows applications to utilize the Jakarta Mail 2.0 API.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.websphere.appserver.features/visibility/public/mail-2.0/resources/l10n/io.openliberty.mail-2.0.properties:17]: Update to

>This feature allows applications to use the Jakarta Mail 2.0 API.

**Review by Charlotte-Holt — COMMENTED** [review]
@uberskigeek Just one quick ID review update. Let me know if you have questions. Thx!

**Comment by skasund** [comment]
L2 message review not required

**Comment by uberskigeek** [comment]
#build

**Comment by uberskigeek** [comment]
This is handled in #14108

**Pull request closed without merging**